message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -215,16 +206,22 @@
   the configuration operation doesn't actually take place."
   identity)
 
+;; parameter is type
 (defmethod create-aws-client :sqs
-  [^Keyword type]
+  [^Keyword _]
```

Comment:
and here _type if you want.  I am no longer going to point these out, even though I see more below.

Reply:
in the future lets just say "yada yada yada..." on the second instance in a long list and we know to keep going.  :)